drivers/serial/uart_native_posix.c

```diff
@@ -395,12 +400,13 @@ static void np_add_uart_options(void)
 		{false, false, true,
 		"attach_uart", "", 'b',
 		(void *)&auto_attach, NULL,
-		"Automatically attach to the UART terminal"},
+		"Automatically attach to the UART terminal, by "
+		"default: '" CONFIG_NATIVE_UART_AUTOATTACH_DEFAULT_CMD "'"},
```

Comment:
why did you move this line?

Reply:
In my opinion, if only this parameter is specified, what is the default command to be executed.

Reply:
The default command is what you can override with auto_attach_cmd. The help message is printed in the same order as listed here. So to me it seems clearer as it was originally.
(If you leave this line where it was I can approve this PR :) )